static/app/components/feedback/feedbackItem/feedbackAssignedTo.tsx

```diff
@@ -0,0 +1,126 @@
+import {useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {fetchOrgMembers} from 'sentry/actionCreators/members';
+import {AssigneeSelectorDropdown} from 'sentry/components/assigneeSelectorDropdown';
+import ActorAvatar from 'sentry/components/avatar/actorAvatar';
+import {Button} from 'sentry/components/button';
+import {
+  EventOwners,
+  getAssignedToDisplayName,
+  getOwnerList,
+} from 'sentry/components/group/assignedTo';
+import {IconChevron, IconUser} from 'sentry/icons';
+import {space} from 'sentry/styles/space';
+import {defined} from 'sentry/utils';
+import type {FeedbackEvent, FeedbackIssue} from 'sentry/utils/feedback/types';
+import useApi from 'sentry/utils/useApi';
+import useCommitters from 'sentry/utils/useCommitters';
+import useOrganization from 'sentry/utils/useOrganization';
+
+interface Props {
+  feedbackEvent: FeedbackEvent | undefined;
+  feedbackIssue: FeedbackIssue;
+}
+
+export default function FeedbackAssignedTo({feedbackIssue, feedbackEvent}: Props) {
+  const organization = useOrganization();
+  const api = useApi();
+  const project = feedbackIssue.project;
+  const [eventOwners, setEventOwners] = useState<EventOwners | null>(null);
+  const {data} = useCommitters(
+    {
+      eventId: feedbackEvent?.id ?? '',
+      projectSlug: project.slug,
+    },
+    {
+      notifyOnChangeProps: ['data'],
+      enabled: defined(feedbackEvent?.id),
+    }
+  );
+
+  useEffect(() => {
+    // TODO: We should check if this is already loaded
+    fetchOrgMembers(api, organization.slug, [project.id]);
+  }, [api, organization, project]);
+
+  useEffect(() => {
+    if (!feedbackEvent) {
+      return () => {};
+    }
+
+    let unmounted = false;
+
+    api
+      .requestPromise(
+        `/projects/${organization.slug}/${project.slug}/events/${feedbackEvent.id}/owners/`
+      )
+      .then(response => {
+        if (unmounted) {
+          return;
+        }
+
+        setEventOwners(response);
+      });
+
+    return () => {
+      unmounted = true;
```

Comment:
Should we cancel the api request on unmount? `api.cancel()` iirc

Reply:
`api.clear()` maybe?